Agent-Install-Examples/eks_lumos_install_python/README.md

```diff
@@ -19,11 +19,14 @@ NOTE: The API `base_url` info can be found when creating the API credentials in
 
 Example of the `config.json` file
 
+NOTE: The`base_url` will be provided when creating the API keys.
+
 ```
 {
         "falcon_client_id": "8972331hiusgdaugd987987231213",
         "falcon_client_secret": "cdHJKASDsdad7t8sdsajhkasdasd",
-        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH"
+        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH",
+        "base_url": "https://api.us-2.crowdstrike.com"
```

Comment:
Are these real client ID, secret, and CIDs?

Reply:
Looked closer and noticed the `asdasd` bits 😌 

Believe there is a semi-standardized string for these we've been using in code and docs. @redhatrises do you have them?

Reply:
For CID, the standardized string is: `1234567890ABCDEF1234567890ABCDEF-12`. Should come up with something for id and secret.

Reply:
I will fix it up